src/components/Card/src/Card/card.module.css

```diff
@@ -66,13 +47,15 @@
     font-size: var(--denhaag-card-subtitle-font-size);
     font-weight: var(--denhaag-card-subtitle-font-weight);
     line-height: var(--denhaag-card-subtitle-line-height);
-    padding-block: var(--denhaag-card-subtitle-padding) 0;
-    padding-inline: var(--denhaag-card-subtitle-padding);
+    padding-block-start: var(--denhaag-card-subtitle-padding);
+    padding-block-end: var(--denhaag-card-subtitle-padding);
+    padding-inline-start: var(--denhaag-card-subtitle-padding);
+    padding-inline-end: var(--denhaag-card-subtitle-padding);
 }
 
 .denhaag-card--case {
   position: relative;
-  background: var(--denhaag-green-2);
+  background: var(--denhaag-card--case-folder-background-color);
```

Comment:
CSS variables don't typically follow the `--` BEM double-dash syntax.

Reply:
Allright, have made some changes to the variable names